pkg/envoy/ads/response_test.go

```diff
@@ -97,30 +96,6 @@ var _ = Describe("Test ADS response functions", func() {
 		Expect(err).ToNot(HaveOccurred())
 	})
 
-	Context("Test makeRequestForAllSecrets()", func() {
```

Comment:
Since some changes here look the same as #4758, please update this branch with the recent changes from `main` (ideally with a merge, not a rebase) so we don't have to review the same changes again.

Reply:
done!